extensions/hibernate-search-orm-elasticsearch/deployment/src/main/java/io/quarkus/hibernate/search/orm/elasticsearch/deployment/HibernateSearchElasticsearchProcessor.java

```diff
@@ -174,6 +178,7 @@ void registerBeans(List<HibernateSearchElasticsearchPersistenceUnitConfiguredBui
     @BuildStep
     @Record(ExecutionTime.STATIC_INIT)
```

Comment:
there's
```java
// Some user-injectable beans are retrieved programmatically and shouldn't be removed
        unremovableBean.produce(UnremovableBeanBuildItem.beanTypes(FailureHandler.class,
                AutomaticIndexingSynchronizationStrategy.class,
                ElasticsearchAnalysisConfigurer.class, IndexLayoutStrategy.class));
```
just above. Do we need to add `IndexingPlanSynchronizationStrategy` to the list ?

Reply:
Good catch! Yes we do.